### ISSUE TITLE
False-positive "Use $(x) instead of $x"

### ISSUE THREAD

**Issue by mlange-42** [RelationalAI-oss/ReLint.jl#66]
ReLint gives false-positive "Use $(x) instead of $x" for this:

```julia
SUITE["benchmark_world_add_remove n=$n"] = @be setup_world_add_remove($n) benchmark_world_add_remove(_, $n) seconds = SECONDS
```

Here, I thought it probably means the "($n)". But this one gives the warning, too:

```julia
data_current = [read_bench_table("bench_current_$i.csv") for i in 1:3]
```

**Comment by kuszmaul** [comment]
I don't see a false positive. The second one is referring to the $i

**Comment by mlange-42** [comment]
Doesn't it tell me I should not use `()` around the variable? How does the linter want me to express this?

**Comment by mlange-42** [comment]
Ok, this was a misunderstanding, as ReLint wanted the opposite in other places in my code. Thanks!
